core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -280,6 +281,16 @@ private[spark] abstract class MapOutputTracker(conf: SparkConf) extends Logging
     }
   }
 
+  protected def supportsContinuousBlockBulkFetch: Boolean = {
+    // continuousBlockBulkFetch only happens in SparkSQL, it uses UnsafeRowSerializer,
+    // which supports relocation of serialized objects, so we only consider compression
+    val adaptiveEnabled: Boolean = conf.getBoolean("spark.sql.adaptive.enabled", false)
```

Comment:
I don't think we can get this conf here. SQL conf is per-session and is kind of a layer above Spark conf.

Reply:
Got it, adaptiveEnabled will be removed as your suggestion.